app/editor/src/features/content/form/ClipDirectoryFilter.tsx

```diff
@@ -0,0 +1,35 @@
+import { IconButton } from 'components/form';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Row, Text } from 'tno-core';
+
+interface IClipDirectoryFilterProps {
+  setGlobalFilter: (filterValue: any) => void;
+}
+
+export const ClipDirectoryFilter: React.FC<IClipDirectoryFilterProps> = ({ setGlobalFilter }) => {
+  const [filter, setFilter] = React.useState<string>('');
+  const navigate = useNavigate();
+  return (
+    <>
+      <Row className="filter-bar" justifyContent="center">
+        <Text
+          onChange={(e) => {
+            setFilter(e.target.value);
```

Comment:
Why do you have two filters?

Reply:
I used one of Jonny's examples as a template. I've removed setFilter() and it works fine.

Reply:
The value of the filter const is used to track the contents of the text box. When you click on the "reset" button, it ensures the text box is emptied.